modules/theme/src/themes/dark/assets/images/icons/building-icon.svg

```diff
@@ -1,5 +1,3 @@
-<?xml version="1.0" encoding="iso-8859-1"?>
```

Comment:
Could you explain the reason to remove this line?

Reply:
Fixed